libs/vkd3d/command.c

```diff
@@ -2662,10 +2663,16 @@ static void vk_access_and_stage_flags_from_d3d12_resource_state(const struct d3d
                 *stages |= VK_PIPELINE_STAGE_DRAW_INDIRECT_BIT;
                 *access |= VK_ACCESS_INDIRECT_COMMAND_READ_BIT;
 
-                if (device->vk_info.EXT_conditional_rendering)
+                /* D3D12_RESOURCE_STATE_PREDICATION */
```

Comment:
Do we never use actual VK_PIPELINE_STAGE_CONDITIONAL_RENDERING_EXT?

Reply:
We do after we copy the predicate to its scratch memory; there's a separate barrier to take care of that.